static/library/map_component.ts

```diff
@@ -146,16 +162,58 @@ export class DatacommonsMapComponent extends LitElement {
   }
 
   render(): HTMLElement {
-    const place = this.parentPlace || this.place || this.placeDcid;
-    const variable = this.variable || this.statVarDcid;
-    const childPlaceType = this.childPlaceType || this.enclosedPlaceType;
+    let dataSpec = [];
+    if (!_.isEmpty(this.parentPlaces) && !_.isEmpty(this.childPlaceTypes)) {
+      this.parentPlaces?.forEach((placeDcid, index) => {
```

Comment:
nit: don't need the "?" check here since you've already checked that this.parentPlaces isn't empty

Reply:
Good call out. Updated.